Doesn't work with Eleventy 2.0.0 canary

Just FYI, the current Eleventy 2.0.0 canary (with @zachleat's new server to substitute for BrowserSync) apparently has a problem with this plugin, since it crashed on launch with the following:

```bash
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: eleventy_site@undefined
npm WARN Found: @11ty/eleventy@1.0.0
npm WARN node_modules/@11ty/eleventy
npm WARN   peer @11ty/eleventy@">=0.10.0-beta.1" from eleventy-plugin-embed-tweet@0.3.1
npm WARN   node_modules/eleventy-plugin-embed-tweet
npm WARN     dev eleventy-plugin-embed-tweet@"^0.3.1" from the root project
npm WARN   1 more (the root project)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer @11ty/eleventy@">=0.10.0-beta.1" from eleventy-plugin-embed-tweet@0.3.1
npm WARN node_modules/eleventy-plugin-embed-tweet
npm WARN   dev eleventy-plugin-embed-tweet@"^0.3.1" from the root project
```

--- a/.eleventy.js
+++ b/.eleventy.js
@@ -1,4 +1,5 @@
-var twitter = require("./twitter")
+const pkg = require("./package.json");
+const twitter = require("./twitter")
 
 module.exports = {
     initArguments: {},
@@ -6,6 +7,13 @@ module.exports = {
         // combine destructured option params
         let options = {cacheDirectory, useInlineStyles, autoEmbed}
         
+        try {
+            eleventyConfig.versionCheck(pkg["11ty"].compatibility);
+        } catch (e) {
+            console.log(
+              `WARN: Eleventy Plugin (${pkg.name}) Compatibility: ${e.message}`
+            );
+        }
         // added in 0.10.0
         eleventyConfig.addNunjucksAsyncShortcode("tweet", async(tweetId) => {
             return await twitter.getTweet(tweetId, options)

--- a/package.json
+++ b/package.json
@@ -26,9 +26,6 @@
         "url": "http://kylemit.dev/"
     },
     "license": "MIT",
-    "peerDependencies": {
-        "@11ty/eleventy": ">=0.10.0-beta.1"
-    },
     "dependencies": {
         "dotenv": "^8.2.0",
         "html-minifier": "^4.0.0",
@@ -37,5 +34,8 @@
         "request": "^2.88.0",
         "request-promise": "^4.2.5",
         "string-replace-async": "^1.2.1"
+    },
+    "11ty": {
+        "compatibility": ">=0.10.0"
     }
 }

